website/content/zh/wgs/platforms/whitepaper/index.md

```diff
@@ -383,7 +383,7 @@ implement and enforce desired practices for security, performance, cost
 governance and consistent experience. The following graphic illustrates the
 relationships between products, platforms, and capability providers.
 
-<img src="assets/platform_components.png" width=600px />
+<img src="assets/platforms-def.drawio.png" width=600px />
```

Comment:
Did you mean to point to the Chinese version here?

Reply:
No, here is still English because the Chinese version has not been merged in another pr #508 .  
We cloud update it after PR 508 merged.

Reply:
Okay then, I'll merge!